JSDemos/Demos/Form/ColumnsAdaptability/description.md

```diff
@@ -1 +1,21 @@
-The Form component can use different layouts depending on&nbsp;the screen width. The dependency between column count and the screen width is&nbsp;specified using the **colCountByScreen** property, which has a&nbsp;higher priority than the **colCount** and the **minColWidth** properties. In&nbsp;this demo, the form is&nbsp;displayed using a&nbsp;4-column layout on&nbsp;wide screens and a&nbsp;2-column layout on&nbsp;narrow screens. Switch between the landscape and portrait layouts to&nbsp;see how the form looks on&nbsp;wide and narrow screens.
\ No newline at end of file
+The Form component can use different layouts depending on the screen width. Click one of the buttons on the right from the demo title to switch between horizontal and vertical orientations. You can determine the number of columns in the Form in one of the following manners:
+
+**Calculate the Number of Columns Automatically**         
+
+To enable automatic calculation of the number of columns based on the screen width, follow these steps:
```

Comment:
Is it necessary to say "enable automatic calculation" here? For example, can we say the following instead?

`To automatically calculate the number of columns based on the screen width, follow these steps:`